src/microbe_stage/organelle_components/MovementComponent.cs

```diff
@@ -49,6 +48,15 @@ protected override void CustomAttach()
         SetSpeedFactor(0.25f);
     }
 
+    protected override bool NeedsUpdateAnyway()
+    {
+        // The basis of the transform represents the rotation, as long as the rotation is not modified,
+        // the organnel needs to be updated.
+        // TO DO:
```

Comment:
two typos here, and TODO: needs to have the TODO comment immediately following it, not on the next line